makefiles/toolchain_gnu-arm-eabi.mk

```diff
@@ -60,7 +60,7 @@ endif
 
 # XXX this is pulled pretty directly from the fmu Make.defs - needs cleanup
 
-MAXOPTIMIZATION		 ?= -O3
+MAXOPTIMIZATION		 ?= -Os
```

Comment:
We can't change the optimization level in general. Please set the optimization options per module in the module.mk. NuttX is already built with -Os